src/transpile.rs

```diff
@@ -539,6 +539,10 @@ impl MutationEntrypoint<'_> for DeleteBuilder {
 
 impl OrderByBuilder {
     fn to_order_by_clause(&self, block_name: &str) -> String {
+        #[cfg(feature = "pg15")]
+        let mut frags = vec![];
+
+        #[cfg(not(feature = "pg15"))]
         let mut frags = vec!["true".to_string()];
```

Comment:
Alternatively, something like this
```suggestion
        let mut frags = vec![];

        #[cfg(not(feature = "pg15"))]
        frags.push("true".to_string());
```